protool/__init__.py

```diff
@@ -23,6 +24,26 @@ class ProvisioningType(Enum):
     ENTERPRISE_DISTRIBUTION = 7
 
 
+class Certificate:
```

Comment:
I just had a play around with this and I'm not convinced there's any real value in having this wrapper. I can see a slight convenience but I'm concerned about a couple of problems:

1. It's an extra layer of indirection for whoever uses it.
2. It's tedious to add new methods that are just wrappers of the `x509` methods.

I think we should just return a list of `x509` certificates instead.

Reply:
Changed from dedicated cert object to list of X509 certificates.